src/app/fdctl/run/tiles/fd_store_int.c

```diff
@@ -0,0 +1,604 @@
+/* Store tile manages a blockstore and serves requests to repair and replay. */
+
+#define _GNU_SOURCE 
+
+#include "tiles.h"
+
+#include "generated/store_int_seccomp.h"
+#include "../../../../flamenco/repair/fd_repair.h"
+#include "../../../../flamenco/runtime/fd_blockstore.h"
+#include "../../../../flamenco/leaders/fd_leaders.h"
+#include "../../../../util/fd_util.h"
+
+#include <unistd.h>
+#include <arpa/inet.h>
+#include <linux/unistd.h>
+#include <sys/random.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+
+#include "../../../../util/net/fd_eth.h"
+#include "../../../../util/net/fd_ip4.h"
+#include "../../../../util/net/fd_udp.h"
+#include "../../../../disco/shred/fd_stake_ci.h"
+#include "../../../../disco/tvu/fd_store.h"
+#include "../../../../disco/keyguard/fd_keyload.h"
+#include "../../../../flamenco/leaders/fd_leaders.h"
+#include "../../../../flamenco/runtime/fd_runtime.h"
+
+#define SHRED_IN_IDX    0
+#define REPAIR_IN_IDX   1
+#define STAKE_IN_IDX    2
+#define REPLAY_IN_IDX   3
+
+#define REPAIR_OUT_IDX  0
+#define REPLAY_OUT_IDX  1
+
+#define MAX_REPAIR_PEERS (40200UL)
```

Comment:
Are these consensus related? Where do these constants come from?

Reply:
Not consensus related